src/Microsoft.OData.Core/UriParser/Binders/InBinder.cs

```diff
@@ -423,5 +424,83 @@ private static string NormalizeDateTimeCollectionItems(string bracketLiteralText
 
             return "[" + String.Join(",", items) + "]";
         }
+
+        private static bool IsCollectionContentEmptyOrSpaces(string bracketLiteralText)
+        {
+            if (string.IsNullOrWhiteSpace(bracketLiteralText) || bracketLiteralText.Length < 2)

```

Comment:
Will validation that the first non-whitespace character is `(` or `[` and the last non-whitespace character is `)` or `]` have been done prior to calling this method?

Asking because this method would return true for the input `(1`

Reply:
This is handled somewhere else. Where we have unbalanced brackets or an incorrect format.

Reply:
Maybe the check here is not necessary.

Reply:
remove the check.